packages/integration/config.d.ts

```diff
@@ -15,41 +15,115 @@
  */
 
 export interface Config {
+  /** Configuration for integrations towards various external repository provider systems */
   integrations?: {
+    /** Integration configuration for Azure */
     azure?: Array<{
-      /** @visibility frontend */
+      /**
+       * The hostname of the given Azure instance
+       * @visibility frontend
+       */
       host: string;
+      /**
+       * Token used to authenticate requests.
+       * @visibility secret
+       */
+      token?: string;
     }>;
 
+    /** Integration configuration for BitBucket */
```

Comment:
```suggestion
    /** Integration configuration for Bitbucket */
```

Contrary to GitHub, GitLab, they use `Bitbucket`. https://bitbucket.org/